### ISSUE TITLE
The Windows 10 Pro step fails.  https://cloud.ibm.com/docs?tab=develop

### ISSUE THREAD

**Issue by duncanmarkc** [ibm-cloud-docs/overview#31]
Did you know you can also submit a pull request to update any IBM Cloud documentation topic by clicking the Edit Topic link on any page? For more information, see Providing a content update with a pull request: https://cloud.ibm.com/docs/overview?topic=overview-feedback#feedback-pr.

1. What are the details of the issue you're experiencing?
[Step 1 Fail.docx](https://github.com/ibm-cloud-docs/overview/files/6865900/Step.1.Fail.docx)

2. Is this issue in reference to a specific page? If so, please provide a link.
https://cloud.ibm.com/docs?tab=develop
3. Do you have a suggestion about how to resolve the issue?

**Comment by christoc-ibm** [comment]
Sent this issue to the CLI dev team.

**Comment by steveclay** [comment]
I'm wondering if the user was using Administrator during the install as that could impact successfully creating the shortcut. Also, although the error message was returned, I can see that the CLI was installed properly as the commands to install the plug-ins were successful. The only piece missing is the `ic` shortcut that points to `ibmcloud`.

**Comment by duncanmarkc** [comment]
From the instructions: For Windows 10 Pro, run the following command as an administrator in PowerShell:

[Net.ServicePointManager]::SecurityProtocol = "Tls12"; iex(New-Object Net.WebClient).DownloadString('https://ibm.biz/idt-win-installer')

**Comment by duncanmarkc** [comment]
Yes, I installed it using Administrator.  From the error you can see that
the install path did not exist.  Is the IBMCLOUD CLI  an application that
installs like Nodejs??  Or is it something I run from a command prompt?
Do the Cloud Functions (CF) plugin install as a plugin to a browser?

Once I get past this I can continue on with my Labs.

Thanks,

Mark Duncan

Mark C. Duncan
Technical Architect
IBM GBS Public Service
***@***.***
757.876.9942




From:	"steveclay" ***@***.***>
To:	"ibm-cloud-docs/overview" ***@***.***>
Cc:	"duncanmarkc" ***@***.***>, "Author"
            ***@***.***>
Date:	07/23/2021 02:49 PM
Subject:	[EXTERNAL] Re: [ibm-cloud-docs/overview] The Windows 10 Pro
            step fails.  https://cloud.ibm.com/docs?tab=develop (#31)



I'm wondering if the user was using Administrator during the install as
that could impact successfully creating the shortcut. Also, although the
error message was returned, I can see that the CLI was installed properly
as the commands to install ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.
ZjQcmQRYFpfptBannerEnd


I'm wondering if the user was using Administrator during the install as
that could impact successfully creating the shortcut. Also, although the
error message was returned, I can see that the CLI was installed properly
as the commands to install the plug-ins were successful. The only piece
missing is the ic shortcut that points to ibmcloud.

**Comment by mekaufman** [comment]
@steveclay Can you help the customer with his latest comment above?

**Comment by mekaufman** [comment]
@duncanmarkc Please follow up with Steve Clay. I provided some details in a private message on slack. Thanks.
